blockchain-scrapers/blockchains/neo/docker-compose.neo.yml

```diff
@@ -7,8 +7,7 @@ services:
     ports:
       - "10332:10332"
     volumes:
```

Comment:
same we prob dont want to expose the port to the outside world.

Reply:
Fixed in the latest commit